dist/mercury.js

```diff
@@ -4076,38 +4087,6 @@ var WwwWesternjournalismComExtractor = {
   }
 };
 
-var FusionNetExtractor = {
-  domain: 'fusion.net',
-  title: {
-    selectors: ['.post-title', '.single-title', '.headline']
-  },
-  author: {
-    selectors: ['.show-for-medium .byline']
-  },
-  date_published: {
-    selectors: [['time.local-time', 'datetime']]
-  },
-  dek: {
-    selectors: [// enter selectors
-    ]
-  },
-  lead_image_url: {
-    selectors: [['meta[name="og:image"]', 'value']]
-  },
-  content: {
-    selectors: [['.post-featured-media', '.article-content'], '.article-content'],
-    // Is there anything in the content you selected that needs transformed
-    // before it's consumable content? E.g., unusual lazy loaded images
-    transforms: {
-      '.fusion-youtube-oembed': 'figure'
-    },
-    // Is there anything that is in the result that shouldn't be?
-    // The clean selectors will remove anything that matches from
-    // the result
-    clean: []
-  }
-};
-
```

Comment:
yeah so the `FusionNetExtractor` extractor is removed here, but still exists in the `dist` folder so we must be missing a build step somewhere?